_sass/_aiml.scss

```diff
@@ -63,4 +68,144 @@
     padding: 3em;
     width: calc(100% - 6em);
     position: relative;
+    flex-shrink: 0.35;
+}
+.page-container{
+    display: flex;
+    justify-content: flex-start;
+}
+.main-content{
+    width: 1000px;
+}
```

Comment:
I think we already have a class that does something similar

Reply:
Yea, I removed this.